shared/schema/templates/Decision.tsx

```diff
@@ -0,0 +1,145 @@
+import {Logger} from '../../render/Logger';
+import {TemplateBodyType, TemplateChild} from './Templates';
+
+export enum Persona {
+  LIGHT = 'light',
+  DARK = 'dark',
+}
+export enum Skill {
+  ATHLETICS = 'athletics',
+  KNOWLEDGE = 'knowledge',
+  CHARISMA = 'charisma',
+}
+export enum Outcome {
+  SUCCESS = 'success',
+  FAILURE = 'failure',
+  RETRY = 'retry',
+  INTERRUPTED = 'interrupted',
```

Comment:
Interrupted feels like there could be a better / shorter word for it if writers will be typing it a lot - miss? Guess it's weird since failure can mean either "failure with penalties" like crit fail, or "failure to succeed" like miss

Reply:
Let's think about this and change it later.